frontend/public/components/_image-stream.scss

```diff
@@ -1,3 +1,49 @@
 .co-example-docker-command__popover {
   width: 600px;
 }
+
+.co-images-stream-tag-timeline {
+  width: auto;
+  margin: 20px 30px;
+  @media (max-width: $screen-xs-max) {
```

Comment:
I prefer these to come after the other declarations so it doesn't interrupt the flow of the declarations.